pypfopt/efficient_frontier.py

```diff
@@ -326,7 +326,7 @@ def efficient_return(self, target_return, market_neutral=False):
         """
         if not isinstance(target_return, float) or target_return < 0:
             raise ValueError("target_return should be a positive float")
-        if target_return > self.expected_returns.max():
+        if target_return > np.abs(self.expected_returns).max():
```

Comment:
@robertmartin8 When shorting is allowed, returns higher than that are also possible.
Continuing your example from #215, when shorting the -10% exp. return asset and investing the proceeds into the 5% exp. return asset, we obtain a portfolio return of -1*(-10%) + 2*5% = 20% (or -1*(-10%) + 1*5% = 15%, if you are in the market neutral setting).